german/Einladen/teufelsrad.md

```diff
@@ -0,0 +1,6 @@
+
+Sie ist neugierig. Sie hat schon vieles über das Teufelsrad gehört.
+Sie hat es aber noch nicht gesehen. Da Sie dich nett und hübsch findet willigt Sie ein, das Teufelsrad mit dir anzuschauen.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.